Sources/Submissions/Tags/InputTag.swift

```diff
@@ -10,6 +10,8 @@ public final class InputTag: TagRenderer {
         let hasErrors: Bool
         let placeholder: String?
         let helpText: String?
+        let accept: String? // for file input
```

Comment:
Since these 2 parameters are exclusive to file input I wonder if this might be a better candidate for its own tag. Just like select or checkbox will have to be because they don't fit well with the other parameters.

Reply:
Done!